src/main/scala/org/broadinstitute/dsde/firecloud/service/ExportEntitiesByTypeActor.scala

```diff
@@ -63,13 +64,19 @@ class ExportEntitiesByTypeActor(rawlsDAO: RawlsDAO,
                                 workspaceName: String,
                                 entityType: String,
                                 attributeNames: Option[IndexedSeq[String]],
+                                model: Option[String],
                                 argMaterializer: ActorMaterializer)
                                (implicit protected val executionContext: ExecutionContext) extends Actor with LazyLogging {
 
   implicit val timeout: Timeout = Timeout(1 minute)
   implicit val userInfo: UserInfo = argUserInfo
   implicit val materializer: ActorMaterializer = argMaterializer
 
+  implicit val modelSchema: ModelSchema = model match {
+    case Some(name) => ModelSchemaRegistry.getModelForSchemaType(SchemaTypes.withName(name))
+    case None => ModelSchemaRegistry.getModelForEntityType(entityType)
```

Comment:
This case allows for the flexible model to be chosen if it's not specified. I think the schema type should be known by this point in the code and should be determined right at the point where we get the optional parameter from the request. That way, if the client doesn't specify a schema, it behaves exactly the same regardless of the entity type.

Reply:
for backwards compatibility, no model specified should be the firecloud model. this is so that existing code does not have to start passing a model param to indicate the "firecloud".

Reply:
Right. So if model isn't specified but the entity type doesn't match then this case will select the flexible model. In prod today, the firecloud model would still be used in that case. Although that usage is unsupported, it does look to me like a change in behavior.

Reply:
ah. got it.